src/backend/db/database_manager.py

```diff
@@ -259,3 +309,91 @@ def check_customer_guid_exists(self, customer_guid):
         finally:
             session.close()
 
+    def add_custom_field(self, customer_guid, field_name, field_type, required):
```

Comment:
We need to change the add custom field, list custom field and delete custom field function according to the new database schema

Reply:
i updated schema as per the new requirements.